### ISSUE TITLE
forkSense Segmentation fault

### ISSUE THREAD

**Issue by rstraver** [MBoemo/DNAscent#72]
Hi again, I'm trying to run forkSense on the bam output (using DNAscent commit 5bc3279) but it seems to crash the moment it needs to write the output file:

```
Processed 20 reads...
./test_scent_fork.sh: line 5: 2016257 Segmentation fault      (core dumped) DNAscent forkSense -d ./test_scent_gpu/few.bam -o ./test_scent_gpu/few.forkSense --order EdU,BrdU
```

Seems the actual reads don't matter (much), I extracted the first < 50 for this test, I get the same issue for a bigger file (>19800 reads): It appears to process all reads, then crash. The output file location is writable, and I tested both with and without a pre-existing (empty) output file.

Any suggestions on what this could be?

**Comment by MBoemo** [comment]
Is `few.bam` small enough that you can send it to me? If so happy to take a look.

**Comment by rstraver** [comment]
Here's the tiny bam file, it's about 33 reads:
[few.bam.zip](https://github.com/user-attachments/files/17458457/few.bam.zip)

**Comment by rstraver** [comment]
Hi @MBoemo, I haven't heard from you for a while. I responded to your e-mail, twice, did you perhaps not receive anything? I'm still waiting for a solution, not sure if this is a difficult to solve issue or something went wrong. 
Let me know if I need to try sending you the data through other means.

**Comment by MBoemo** [comment]
Should now be fixed in 1432cf75f1e57920856f1ca465ad3bc5650ebbe9 (and singularity image has been updated with the fix).
